core/contracts/financial_templates/PricelessPositionManager.sol

```diff
@@ -197,21 +216,87 @@ contract PricelessPositionManager is Testable {
         tokenCurrency.burn(numTokens.rawValue);
     }
 
+    /**
+     * @dev This overrides pfc() so the PricelessPositionManager can report its profit from corruption.
+     */
+
+    function pfc() public view returns (FixedPoint.Unsigned memory) {
```

Comment:
This is great! However I'm not sure if this value of `pfc` considers the current pending liquidations in `Liquidatable.sol`, does it? When a liquidation is initiated the underlying is decremented within `PricelessPositionManager` and so any pending liquidations are not considered within the profit from corruption. To solve this we might need to track the current underlying that is pending liquidation and include this in the total `pfc` calculation for an `ExpiringMultiParty`.

Reply:
Yep, exactly. I wanted to start with the priceless position manager before moving on to Liquidatable, since pfc and fees should be deducted from both (although I hope we can keep the logic separate). I should've noted that in the PR description. Will add now.